incubator/busbar/templates/deployment.yaml

```diff
@@ -0,0 +1,171 @@
+---
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "busbar.fullname" . }}-web
+  labels:
+    app: {{ template "busbar.fullname" . }}-web
```

Comment:
The `app` label should be `{{ template "busbar.name" . }}` on all resources the chart creates.